modules/ttdBidAdapter.js

```diff
@@ -58,7 +58,9 @@ function getBidFloor(bid) {
 }
 
 function getSource(validBidRequests) {
-  let source = {};
+  let source = {
+    tid: validBidRequests[0].transactionId
```

Comment:
@JudSpencer mentioned in ortb slack he would prefer if you guys followed Rob's proposal and implementation here https://github.com/prebid/Prebid.js/pull/8641/files to gather imp.ext.tid

Reply:
adding to this, source.tid as the first transaction id makes sense to me, but we also seem to have debate over whether it should be the auction id, and implementors have chosen one or the other. Prebid Server seems to just make up its own id. So aside from the cardinality issue that imp.ext.tid addresses, we have other issues with adapters defining it differently (we have open issues discussing). Imp.ext.tid shares none of these concerns, only adoption, hence me encouraging you to pick it up here.

Reply:
I updated this but still new to the area so please verify it is what you expected.